### PR TITLE
Edit: Remove usless lines and some improvment

### PR DESCRIPTION
**Pull request by zakariaGatter**
i check the layouts folder and i add some improvment

### CODE REVIEW

File: src/utils/layout.sh [hunk 1]
```diff
@@ -10,9 +10,7 @@ rotate() {
   have=${have:1:${#have}-2}
   angle=$3
 
-  if [[ "$have" != "$want" ]]; then
-    bspc node "$node" -R "$angle"
-  fi
+  [ "$have" = "$want" ] || bspc node "$node" -R "$angle"
```
Comment (amtoine) [inline src/utils/layout.sh:13]: i find `[ "$have" != "$want" ] &&` clearer than `[ "$have" = "$want" ] ||`, but that's quite a personal feeling :yum:

**Review by amtoine — COMMENTED** [review]
Thanks for the work :star_struck: 

everything looks great, waiting for @phenax' review :wink: 
just one minor remark :+1:

File: src/utils/layout.sh (hunk 1 above)
Comment (zakariaGatter) [reply src/utils/layout.sh:13]: Yes me too. I just change cause is feel wrong to me but ill change it back 👍🏻

**Review by amtoine — COMMENTED** [review]
thanks for the changes on the implicit `if` statement :relieved:

File: src/utils/layout.sh (hunk 1 above)
Comment (amtoine) [reply src/utils/layout.sh]: :+1:

**Review by phenax — APPROVED** [review]
Looks good!

**Pull request merged**